go.mod

```diff
@@ -2,4 +2,6 @@ module github.com/edgexfoundry/app-service-configurable
 
 go 1.13
 
-require github.com/edgexfoundry/app-functions-sdk-go v1.0.0
+require github.com/edgexfoundry/app-functions-sdk-go v1.0.1-dev.21
+
+replace github.com/edgexfoundry/app-functions-sdk-go => ../app-functions-sdk-go
```

Comment:
Whoops -- dont want this here

Reply:
oops, forgot to push the update to remove the replace. Fixed.